languages/rust/exercises/concept/vec-macro/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+/// vec-macro for creating empty vectors, and vectors of particular size, with vec!
+
+/// Create an empty vector
+fn create_empty() -> Vec<u8> {
+    unimplemented!()
+}
+
+/// Create a buffer of `count` zeroes.
+///
+/// Applications use these for filling and modifying data
+fn create_buffer(count: usize) -> Vec<u8> {
+    unimplemented!()
+}
+
+/// Create a vector containing the first five digits of the Fibonacci sequence.
+///
+/// Fibonacci's sequence is the list of numbers where the next number is a sum of the previous two.
+fn fibonacci() -> Vec<u8> {
```

Comment:
We should copy the additional comment here from `.meta/example.rs`; I accidentally suggested it there but not here.

Reply:
Nice catch! I copied the file from `example.rs` and replayed the `unimplemented!()` change. Seemed quicker and I might script this to avoid in future PRs 😄